tracer/src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -72,6 +72,11 @@
     <None Remove="AppSec\Waf\ConfigFiles\apisecurity-config.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="..\..\build\_build\CodeGenerators\InstrumentationCategory.cs" Link="ClrProfiler\InstrumentationCategory.cs" />
+    <Compile Include="..\..\build\_build\CodeGenerators\TargetFrameworks.cs" Link="ClrProfiler\TargetFrameworks.cs" />
+  </ItemGroup>
```

Comment:
What's the reason to do this in `Datadog.Trace.csproj`?

Reply:
In the opposite way (files in Datadog.Trace and link in _build) it failed in Linux with a File not found 🤷‍♂️

Reply:
> In the opposite way (files in Datadog.Trace and link in _build) it failed in Linux with a File not found 🤷‍♂️

Urgh, I'd really prefer we don't do this 😟 We get into issues of chasing the files around, esp as the build won't explicitly fail with "file not found" or anything.

The reason it fails is because [in the docker file](https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/build/_build/docker/debian.dockerfile#L75-L78) we copy _only_ the build files across first, and compile the `_build` project.

Does the build _really_ need direct references to these files? 🤔 Instead of having a direct reference, I wonder if we can grab the values from the dlls we load even? 

Worst case, I'd probably actually prefer we just duplicated the files and added a comment 🙈

Reply:
I'm duplicating the files and add comments.